client/asset/dcr/spv.go

```diff
@@ -95,20 +94,31 @@ type dcrWallet interface {
 	GetAllTSpends(ctx context.Context) []*wire.MsgTx
 	TSpendPolicy(tspendHash, ticketHash *chainhash.Hash) stake.TreasuryVoteT
 	VSPHostForTicket(ctx context.Context, ticketHash *chainhash.Hash) (string, error)
-	SetAgendaChoices(ctx context.Context, ticketHash *chainhash.Hash, choices ...wallet.AgendaChoice) (voteBits uint16, err error)
+	SetAgendaChoices(ctx context.Context, ticketHash *chainhash.Hash, choices map[string]string) (voteBits uint16, err error)
 	SetTSpendPolicy(ctx context.Context, tspendHash *chainhash.Hash, policy stake.TreasuryVoteT, ticketHash *chainhash.Hash) error
 	SetTreasuryKeyPolicy(ctx context.Context, pikey []byte, policy stake.TreasuryVoteT, ticketHash *chainhash.Hash) error
 	SetRelayFee(relayFee dcrutil.Amount)
 	GetTicketInfo(ctx context.Context, hash *chainhash.Hash) (*wallet.TicketSummary, *wire.BlockHeader, error)
 	ListSinceBlock(ctx context.Context, start, end, syncHeight int32) ([]walletjson.ListTransactionsResult, error)
-	vspclient.Wallet
+	UnlockOutpoint(txHash *chainhash.Hash, index uint32)
+	SignTransaction(ctx context.Context, tx *wire.MsgTx, hashType txscript.SigHashType, additionalPrevScripts map[wire.OutPoint][]byte,
+		additionalKeysByAddress map[string]*dcrutil.WIF, p2shRedeemScriptsByAddress map[string][]byte) ([]wallet.SignatureError, error)
+	AgendaChoices(ctx context.Context, ticketHash *chainhash.Hash) (choices map[string]string, voteBits uint16, err error)
+	NewVSPTicket(ctx context.Context, hash *chainhash.Hash) (*wallet.VSPTicket, error)
 	// TODO: Rescan and DiscoverActiveAddresses can be used for a Rescanner.
+
+	// VSP client methods
+	SetPublished(ctx context.Context, hash *chainhash.Hash, published bool) error
+	AbandonTransaction(ctx context.Context, hash *chainhash.Hash) error
+	SignMessage(ctx context.Context, msg string, addr stdaddr.Address) (sig []byte, err error)
+	CreateVspPayment(ctx context.Context, tx *wire.MsgTx, fee dcrutil.Amount,
+		feeAddr stdaddr.Address, feeAcct uint32, changeAcct uint32) error
```

Comment:
I commented these out and everything still worked fine. What's the purpose of adding these to the interface?

Reply:
Yeah I'm sorry. I was planning to add an interface over in dcrwallet. Not needed now so removing.

Reply:
Removed.